### DISCUSSION TITLE
Fork android-lbry to remove blacklist

### DISCUSSION THREAD

**Discussion by Tank-Missile** [paveloom-f/lbry-desktop#22] [Ideas]
I'm glad this fork exists to remove the blacklist and allow unrestricted access to all lbry content, but the android app also uses the same blacklist. How hard would it be to remove it from the android app as well?

**Comment by paveloom** [comment]
Well, should be doable if filtering happens in the client itself. I just had a look at the code and I think I found it.

I will get back to it when I have some time, though.

**Comment by paveloom** [comment]
[Bam](https://github.com/paveloom-f/lbry-android)! It's done. Some testing would be appreciated, though.
